src/components/UI/Text/Textarea.vue

```diff
@@ -220,5 +233,32 @@ export default {
     },
   },
   emits: ["input", "keypress", "keydown", "update:value", "start-icon-selected"],
+  // quilljs editor
+  mounted() {
+    this.editor = new Quill(this.$refs.editor, {
+      modules: {
+        toolbar: [["bold", "italic", "underline"]],
+      },
+      theme: "snow",
+      formats: ["bold", "underline", "italic"],
+    });
+
+    this.editor.root.innerHTML = this.value;
+
+    this.editor.on("text-change", () => this.update());
+  },
 };
 </script>
+<style>
+.ql-toolbar.ql-snow {
+  z-index: 5;
+  position: absolute;
+  width: 100%;
+  background-color: #f2e8df;
+  border: 1px solid transparent;
+  padding: 0px;
+}
+.ql-editor {
+  overflow-y: auto;
+}
+</style>
```

Comment:
is this still needed? Because I didn't see these classes being used anywhere

Reply:
Yes, it is required. This css is used by the buttons (bol, itlic, and underline). Except that the `ql-editor` css has yet to be applied.